pb/Client.pb.go

```diff
@@ -40,10 +39,6 @@ const (
 	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
 )
 
-// This is a compile-time assertion that a sufficiently up-to-date version
-// of the legacy proto package is being used.
-const _ = proto.ProtoPackageIsVersion4
```

Comment:
I find it somewhat suspicious that this went away. I noticed the code seems to have been generated with newer versions of protoc & proto-gen-go so that's probably why, but it's curious.